### PR TITLE
Fix toggle-button.

### PR DESCRIPTION
**Pull request by p0psicles**
* Remove the medusa-button wrapper, as it wasn't supposed to be in there at all.

- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: themes-default/slim/views/layouts/main.mako [hunk 1]
```diff
@@ -145,6 +145,7 @@
                     routes
                 });
             }
+            Vue.use(window['vue-js-toggle-button'].default);
```
Comment (p0psicles) [inline themes-default/slim/views/layouts/main.mako]: @OmgImAlexis should I import here?

Comment (sharkykh) [reply themes-default/slim/views/layouts/main.mako]: If you ask me, the component is already imported on line 120, and I see no reason not to put this here.

Comment (OmgImAlexis) [reply themes-default/slim/views/layouts/main.mako]: Yeah Vue is a global so you only need it once per page, if it's in the `main.mako` once it'll be on every single Vue instance on that page. No need to re-register it.

Comment (OmgImAlexis) [reply themes-default/slim/views/layouts/main.mako]: @sharkykh if the imported script doesn't auto install then that line will still be needed.

**Pull request merged**